pypeclub/OpenPype
# Nuke: double slate

## Brief description
This changes a way to merge slate with review. Now it reencodes, so all apps can read the result with slate correctly.
Also, reviews with audio and slate now have audio again.

I tested on op-atom MXF, DNxHD mov, Prores mov, H264 in mp4. With audio (except MXF) and with two ExtractReviewDataMov Output Definitions. Local publish, clean working folder.

## Testing notes:
1. Make a slated output with and with no audio as MXF and H264 in mp4
2. Compare lengths of the sequence output with MFX and mp4 in Resolve
3. See that Resolve reads all frames correctly
